Adafruit_GFX.h

```diff
@@ -163,6 +168,18 @@ class Adafruit_GFX : public Print {
   /**********************************************************************/
   void cp437(boolean x=true) { _cp437 = x; }
 
+ /**********************************************************************/
+  /*!
+    @brief  Enable (or disable) UTF-8-compatible charset in custom made fonts
+            By default, the font.h files boundled to the library use ASCII charset
+	    as UTF-8 fonts need more memory as many AVR chips can offer.
+            Pass 'true' to this function if you are willing to use UTF-8 fonts that
+            you generated whith fontconvert and also your board has enough free memory.
+    @param  x  true = enable (new behavior), false = disable (old behavior)
+  */
+  /**********************************************************************/
+  void utf8(boolean x=true) { _utf8 = x; }
```

Comment:
Seems i was not paying enough attention while copy&pasting code...
Of course this should default to false to prevent creating headache on lower-end boards after the update.